source/common/network/io_socket_handle_impl.cc

```diff
@@ -477,6 +481,15 @@ Api::SysCallIntResult IoSocketHandleImpl::setBlocking(bool blocking) {
   return Api::OsSysCallsSingleton::get().setsocketblocking(fd_, blocking);
 }
 
+IoHandlePtr IoSocketHandleImpl::duplicate() {
```

Comment:
Please implement in the `TestIoSocketHandle` class.

Reply:
fixed in f5a498ac1ff325c37863fe696b81cd1f9850ba9c